packages/kit/src/core/postbuild/fork.js

```diff
@@ -0,0 +1,72 @@
+import { fileURLToPath } from 'node:url';
+import child_process from 'node:child_process';
+
+/**
+ * @template T
+ * @template U
+ * @param {string} module
+ * @param {(opts: T) => U} callback
+ * @returns {(opts: T) => Promise<U>}
+ */
+export function forked(module, callback) {
+	if (process.env.SVELTEKIT_FORK) {
+		process.on(
```

Comment:
I thought you said we shouldn't do this: https://github.com/sveltejs/kit/pull/8636#issuecomment-1403816523

Reply:
i did, but there isn't really a good alternative that doesn't involve a lot more faffery. by checking the `type` and the `module` we effectively guarantee that the only way we could get a false positive is if someone was deliberately trying to trip themselves up